services/hanke-service/src/test/kotlin/fi/hel/haitaton/hanke/domain/HankeTest.kt

```diff
@@ -2,10 +2,7 @@ package fi.hel.haitaton.hanke.domain
 
 import assertk.assertThat
 import assertk.assertions.isEqualTo
-import fi.hel.haitaton.hanke.Haitta13
-import fi.hel.haitaton.hanke.KaistajarjestelynPituus
-import fi.hel.haitaton.hanke.TZ_UTC
-import fi.hel.haitaton.hanke.TodennakoinenHaittaPaaAjoRatojenKaistajarjestelyihin
+import fi.hel.haitaton.hanke.*
```

Comment:
Please configure your IDE to not use wildcard imports.

Reply:
Done